apps/user-office-frontend/src/components/questionary/questionaryComponents/ProposalBasis/QuestionaryComponentProposalBasis.tsx

```diff
@@ -45,18 +46,21 @@ function QuestionaryComponentProposalBasis(props: BasicComponentProps) {
 
   const [localTitle, setLocalTitle] = useState(state?.proposal.title);
   const [localAbstract, setLocalAbstract] = useState(state?.proposal.abstract);
+  const { user } = useContext(UserContext);

```

Comment:
If the logged in user is not used anymore I think it could be completely removed from here.